### PR TITLE
Update Aruba.it

### PR DESCRIPTION
**Pull request by afelicioni**

### CODE REVIEW

**Comment by afelicioni** [comment]
Posted extra commit with updated versioning status

File: _data/hosts.yml [hunk 1]
```diff
@@ -70,13 +70,9 @@
     name: Aruba.it
     url: 'http://hosting.aruba.it/'
     type: shared
-    default: 5.5.28
+    php70: 0
```
Comment (philsturgeon) [inline _data/hosts.yml]: This php70 is not meant to be here. If you saw one anywhere else its an artifact of a merge that I didn't notice.

If you offer PHP 7.0 would you add it to the main versions array below?

Comment (afelicioni) [inline _data/hosts.yml]: Thanks for the note. 

PHP 7.0 is not offered in Aruba right now, going to push a fix by removing the offending line.

**Comment by afelicioni** [comment]
Meanwhile a conflict risen, better let this PR die and open a new one with ultimate data struct?

**Comment by philsturgeon** [comment]
Ok! If you can rustle up some phpinfo URLs too that would help a lot. 

--
Phil Sturgeon

**Comment by afelicioni** [comment]
I would be cool.
Sadly I am not an employee, and I can't provide reachable endpoints for phpinfo options currently available.

**Comment by philsturgeon** [comment]
Ahh, I never know so I always ask. 

Manually updating is ok :)

**Pull request closed without merging**